NEW: Custom-ip (#71)
FalixNodes-Software/KB-articles: content/_posts/minecraft/java/2025-07-15-custom-ip.md

---
title: How To Connect A Custom Domain To Your Minecraft Server
tags: General
permalink: /minecraft/java/general/custom-ip
description: Learn how to add your own custom domain to your minecraft server.
keywords:
    - keyword: IP
      matches: &IP_matches ["Custom", "own"]
    - keyword: domain
      matches: *IP_matches
author:
    - TWIXhunter
    - Korbs
---

{: .warning }
> This article requires you to already own a domain

There are many of places where you can register a domain. Here are a few popular ones:

- [Namecheap](https://namecheap.com/)
- [GoDaddy](https://godaddy.com)
- [Bluehost](https://bluehost.com/domains)
- [Squarespace](https://domains.squarespace.com/)
- [Hover](https://www.hover.com/)

> We are not affiliated with the services mentioned above and are simply providing popular examples. Feel free to use any domain provider you want.

### Creating the Records

> 1. Open the dashboard of your domain registrar.

Reviewer: Because it looks better?????

> 1. Open the dashboard of your domain registrar.

Author: If you think that bold text loosen better for the KB then you should contact Casper to see if he can make all the text bold by default, just making 1 article completely bold isn't going to work

> 1. Open the dashboard of your domain registrar.

Author: And even making all articles bold isn't a good idea beacuse then you wouldn't be able to make things that actually should be bold bolder then the rest of the text

    > The dashboard may look different depending on the registrar. If you need help navigating, please contact the registrar's support team.

2. Navigate to the DNS menu. This menu may also be referred to as "Advanced DNS" or "Domain Records".

> 3. Create a new A-record.

Reviewer: 😱😱😱 wow same as above

> 4. Enter the following info:

Author: See above

> 4. Enter the following info:

Reviewer: I don't need to see above

> **Record Type**:

Reviewer: ```suggestion
**Record Type:**
```

`A-record`

> Setting This value may not be necessary if you selected the record type when creating a new record.

Reviewer: ```suggestion
> (Some registrars may set this automatically when you choose to add a new record.)
```

> **Name** or **Host**:

Reviewer: ```suggestion
**Name** or **Host:**
```

> If you want players to join using `mydomain.tld`, then add: `@`

Reviewer: ```suggestion
- To use `mydomain.tld` as your server address, enter: `@`
```

> 

Reviewer: ```suggestion
```

> If you want players to join using a subdomain (like `play.mydomain.tld` or `mc.mydomain.tld`), then add the subdomain as host (for example: `play`).

Reviewer: ```suggestion
- To use a subdomain (like `play.mydomain.tld` or `mc.mydomain.tld`), enter just the subdomain (e.g., `play`).
```

> **Value**:

Reviewer: ```suggestion
**Value:**
```

> The IPv4 address of your server. This can be found on [the network page](https://client.falixnodes.net/server/network) of your server (Remove the port number and `:` from the end).

Reviewer: ```suggestion
- Enter the **IPv4 address** of your Minecraft server.  
  You can find it on your server's [Network Page](https://client.falixnodes.net/server/network).  
  > Make sure to remove the port and `:` at the end of the address.
```

> **TTL**:

Reviewer: ```suggestion
**TTL:**
```

> Either leave this as `automatic` or enter `30 min`

Reviewer: ```suggestion
- Set this to `Automatic` or `30 min`.
```

> 5. Save the new record and create a new SRV-record.

Reviewer: ```suggestion
5. **Save the A-record** and then **create a new SRV-record**.
```

> 6. Enter the following info:

Reviewer: ```suggestion
6. **Enter the following information for the SRV-record:**
```

**Record Type**:
`SRV-record`

> Setting This value may not be necessary if you selected the record type when creating a new record.

Reviewer: ```suggestion
> (Again, some registrars set this automatically.)
```

**Service**: 
`_minecraft`

**Protocol**:

> If you want people to join using `mydomain.tld`, then add: `_tcp`

Reviewer: ```suggestion
- If using `mydomain.tld`, enter: `_tcp`
```

> 

Reviewer: ```suggestion
```

> If you want players to join using a subdomain (like `play.mydomain.tld` or `mc.mydomain.tld`), then add the subdomain at the end (for example: `_tcp.play`).

Reviewer: ```suggestion
- If using a subdomain (e.g., `play.mydomain.tld`), enter: `_tcp.play`
```

**Priority**:
`0`

**Weight**:
`5`

**Port**:

> The 5 digit value at the end of the IPv4 address from [the network page](https://client.falixnodes.net/server/network), do not enter the `:`.

Reviewer: ```suggestion
Enter the **5-digit port number** from your server’s IP address (found on the [Network Page](https://client.falixnodes.net/server/network)).  
> **Do not** include the `:`.
```

**Target**:

> The domain you want players to use when they join, for example `play.mydomain.tld`.

Reviewer: ```suggestion
The domain or subdomain you want players to connect with, e.g., `play.mydomain.tld`.
```

**TTL**:

> Either leave this as `automatic` or enter `30 min`.

Reviewer: ```suggestion
`Automatic` or `30 min`
```

> 7. Save the new record, 

Reviewer: ```suggestion
7. **Save the SRV-record.**
```

  {: .warning }

>   > Be aware, it might take some time (up to 30 minutes) for your changes to be accepted globally.

Reviewer: ```suggestion
  > Please note: DNS changes may take up to 30 minutes to propagate globally.
```

- To use a subdomain (like `play.mydomain.tld` or `mc.mydomain.tld`), enter just the subdomain (e.g., `play`).

**Value**:
- Enter the IPv4 address of your Minecraft server.  
  You can find it on your server's [Network Page](https://client.falixnodes.net/server/network).  
  > Make sure to remove the port and `:` at the end of the address.

**TTL**:
`Automatic` or `30 min`.

5. Save the A-record and then create a new SRV-record.

6. Enter the following information for the SRV-record:

**Record Type**:
`SRV-record`
    > (Again, some registrars set this automatically.)

**Service**: 
`_minecraft`

**Protocol**:
If you want people to join using `mydomain.tld`, then add: `_tcp`
- If using a subdomain (e.g., `play.mydomain.tld`), enter: `_tcp.play`

**Priority**:
`0`

**Weight**:
`5`

**Port**:
Enter the 5-digit port number from your server’s IP address (found on the [Network Page](https://client.falixnodes.net/server/network)).  
    > **Do not** include the `:`.

**Target**:
The domain or subdomain you want players to connect with, e.g., `play.mydomain.tld`.

**TTL**:
Either leave this as `automatic` or enter `30 min`.

7. Save the SRV-record.

> 

Reviewer 2: ```suggestion

### (Optional) Create a CNAME Record

This step is optional and applies if you prefer to point your custom domain to your existing `.falixsrv.me` subdomain rather than a direct IP address.

**Record Type**:
`CNAME-record`

**Name / Host**:  
Specify the subdomain you wish to use — for example, enter `play` to create `play.mydomain.tld`.

**Target**:  
Enter the subdomain provided by us (e.g., `yourserver.falixsrv.me`).

**TTL**:  
Set to `Automatic` or `30 minutes`, depending on registrar options.

Once completed, `save the CNAME record` to apply the changes.

> Using a CNAME simplifies future management, as you won't need to update the IP manually if it changes, only the target subdomain needs to stay valid.
```

Author: I thought that the Cname record was not needed if you already have an A record

Author: maybe we should replace the A record with the Cname...

Author: although, I have heard that minecraft does weird stuff with Cnames

Author: what about:
An SRV with the target set to the falixsrv.me subdomain
A Cname for the Tlauncher players who don't follow Cname records for some reason

    {: .warning }

>      > Please note: DNS changes may take up to 30 minutes to propagate globally.

Reviewer 2: ```suggestion
     > Please note: DNS changes may take up to 30 minutes to propagate globally.
     > The IP address assigned to your server may change over time. Make sure to update your domain records to match the latest IP shown on the **Network** page.
```
